com.sap.cloud.lm.sl.cf.database/src/test/java/com/sap/cloud/lm/sl/cf/database/migration/executor/type/DatabaseTypeSetterFactoryTest.java

```diff
@@ -0,0 +1,90 @@
+package com.sap.cloud.lm.sl.cf.database.migration.executor.type;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DatabaseTypeSetterFactoryTest {
+
+    private static final String NON_EXISTING_DATABASE_TYPE_SETTER = "test";
+    private static final String BOOL_DEFAULT_DATABASE_TYPE_SETTER = "bool";
```

Comment:
Rename these constants to `BOOL_TYPE`, `LONG_TYPE`, etc. Right now, based on their name, it looks like they should contain instances of `DatabaseTypeSetter`.